bin/ff_map

```diff
@@ -94,14 +94,14 @@ def main():
                 l.append('\n')
         if len(l) > 0:
             lines = [item for sublist in l for item in sublist]
```

Comment:
this line we probably can do more efficient but as long as it's not super slow fine by me

Reply:
I don't notice any huge loss in performance doing this, can you think of a more efficient way?

Reply:
as I said it's fine as long as the files don't get super large